src/SWP/Bundle/MultiTenancyBundle/Initializer/PHPCRBasePathsInitializer.php

```diff
@@ -131,6 +132,7 @@ private function createBasePaths(SessionInterface $session, array $basePaths, ar
                 $route = new $this->documentClass();
                 $route->setParentDocument($this->dm->find(null, $path));
                 $route->setName($home);
+                $route->setType(RouteInterface::TYPE_CONTENT);
```

Comment:
but `Page` implements `RouteInterface` so adding it to that interface will solve issue. I see also that Page don't implement for now required by interface methods. Will add it.